aws/logs_monitoring/parsing.py

```diff
@@ -331,6 +331,13 @@ def find_cloudwatch_source(log_group):
     if log_group.startswith("/aws/appsync/"):
         return "appsync"
 
+    # e.g. /aws/OpenSearchService/domains/*/SEARCH_SLOW_LOGS
+    # e.g. /aws/OpenSearchService/domains/*/INDEX_SLOW_LOGS
+    # e.g. /aws/OpenSearchService/domains/*/ES_APPLICATION_LOGS
+    # e.g. /aws/OpenSearchService/domains/*/AUDIT_LOGS
+    if log_group.startswith("/aws/opensearchservice/domains/"):
+        return "elasticsearch"
```

Comment:
I'm not sure if the returned source should be `elasticsearch` or `opensearch`. I know from [reading the docs](https://docs.datadoghq.com/integrations/elastic/?s=elasticsearch#overview) Datadog knows how to parse ElasticSearch logs (assuming when `source:elasticsearch` is set on a log event) but I'm not sure if OpenSearch logs are different enough to need to parse differently (and if Datadog has some out of the box parsing rules for `source:opensearch`.